From dc0a575ff6a58c39fb8cd5be1702e1b92c479e3a Mon Sep 17 00:00:00 2001
Message-Id: <dc0a575ff6a58c39fb8cd5be1702e1b92c479e3a.1343915404.git.minovotn@redhat.com>
In-Reply-To: <5b3bcf18e866636f24ca1c23eb94d5caf23c6dd6.1343915404.git.minovotn@redhat.com>
References: <5b3bcf18e866636f24ca1c23eb94d5caf23c6dd6.1343915404.git.minovotn@redhat.com>
From: Markus Armbruster <armbru@redhat.com>
Date: Fri, 27 Jul 2012 12:41:50 +0200
Subject: [PATCH 7/7] qemu-keymaps: Finnish keyboard mapping broken

RH-Author: Markus Armbruster <armbru@redhat.com>
Message-id: <1343392910-21593-2-git-send-email-armbru@redhat.com>
Patchwork-id: 40439
O-Subject: [RHEL-6.4 PATCH qemu-kvm 1/1] qemu-keymaps: Finnish keyboard mapping broken
Bugzilla: 794653
RH-Acked-by: Amos Kong <akong@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>

From: Michael Tokarev <mjt@tls.msk.ru>

As mentioned in http://bugs.debian.org/660154 , finnish keyboard mapping
is kind of broken.  Fix it as Timo Sirainen suggests in #660154.

Signed-off-by: Michael Tokarev <mjt@tls.msk.ru>
Signed-off-by: Stefan Hajnoczi <stefanha@linux.vnet.ibm.com>
(cherry picked from commit 0ef62ccd010b835858e81412e6c0cd52c60950dd)
Signed-off-by: Markus Armbruster <armbru@redhat.com>
---
 pc-bios/keymaps/fi |    2 --
 1 file changed, 2 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 pc-bios/keymaps/fi |    2 --
 1 file changed, 2 deletions(-)

diff --git a/pc-bios/keymaps/fi b/pc-bios/keymaps/fi
index 2a4e0f0..4be7586 100644
--- a/pc-bios/keymaps/fi
+++ b/pc-bios/keymaps/fi
@@ -99,9 +99,7 @@ asterisk 0x2b shift
 acute 0x2b altgr
 multiply 0x2b shift altgr
 guillemotleft 0x2c altgr
-less 0x2c shift altgr
 guillemotright 0x2d altgr
-greater 0x2d shift altgr
 copyright 0x2e altgr
 leftdoublequotemark 0x2f altgr
 grave 0x2f shift altgr
-- 
1.7.10.4

